### ISSUE TITLE
VS 2015 RC

### ISSUE THREAD

**Issue by davidtme** [dotnet/efcore#2123]
I’ve been trying to build & test run on the dev branch EntityFramework@c66035fc400c6bae42dc36deefa7bb4568f57a0a in Visual Studio Professional 2015 RC but there seems to be something I'm missing.

Here are the first steps and errors (following the wiki):

Run `@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/release/kvminstall.ps1'))}"` This results in dnx-clr-win-x86.1.0.0-beta5-11706 being installed

Clone the repo git `clone https://github.com/aspnet/EntityFramework.git`

`build initialize` all seems to be ok

`build` Results in the following error:

```
Using Package dependency System.Runtime.Handles 4.0.0-beta-22904
Source: C:\Users\xxx\.dnx\packages\System.Runtime.Handles\4.0.0-beta-22904
File: lib\contract\System.Runtime.Handles.dll

C:\Dev\EntityFramework\src\EntityFramework.Core\Utilities\IndentedStringBuilder.cs(66,37): error CS0246: The type or namespace name 'StringReader' could not be found (are you missing a using directive or an assembly reference?)

Building EntityFramework.Core for .NETPortable,Version=v4.5,Profile=Profile7
Using Project dependency EntityFramework.Core 7.0.0-t150506140348
Source: C:\Dev\EntityFramework\src\EntityFramework.Core\project.json
```

I comment out the block of code and I get a working build but I can't seem to get the test to show up using the build-in vs runner.

`build` on the command line works and starts running tests but an unrelated issues is (looks like a US/UK date mix up):

```
    Microsoft.Data.Entity.Commands.Utilities.CSharpHelperTest.Literal_works_when_DateTime [FAIL]
    Assert.Equal() Failure
                              ↓ (pos 16)
    Expected: DateTime.Parse("3/12/2015 12:00:00 AM")
    Actual:   DateTime.Parse("12/03/2015 00:00:00")
                              ↑ (pos 16)
    Stack Trace:
    Utilities\CSharpHelperTest.cs(60,0): at Microsoft.Data.Entity.Commands.Utilities.CSharpHelperTest.Literal_works(Object value, String expected)
    Utilities\CSharpHelperTest.cs(89,0): at Microsoft.Data.Entity.Commands.Utilities.CSharpHelperTest.Literal_works_when_DateTime()
```

**Comment by bricelam** [comment]
Is build fixed now? (There's a lot of churn underneath EF that can result in intermittent failures)
For tests, our team uses TestDriven.NET.
The culture issues should be fixed now thanks to #2157

**Comment by bricelam** [comment]
Closing for now. Feel free to repoen with additional information.
